index.js

```diff
@@ -11,29 +11,146 @@ const CLIENT_INFO = `${sdkPkg.name}/${sdkPkg.version}`;
 const ENV_INFO = `${seleniumPkg.name}/${seleniumPkg.version}`;
 const utils = require('@percy/sdk-utils');
 const { DriverMetadata } = require('./driverMetadata');
+const log = utils.logger('selenium-webdriver');
+
+const CDP_SUPPORT_SELENIUM = seleniumPkg.version && !isNaN(seleniumPkg.version.charAt(0)) && parseInt(seleniumPkg.version.charAt(0), 10) >= 4;
+
+const getWidthsForMultiDOM = (widths, eligibleWidths) => {
+  let userPassedWidths = [];
+  if (widths.length !== 0) userPassedWidths = widths;
+
+  // Deep copy of eligible mobile widths
+  let allWidths = [];
+  if (eligibleWidths?.mobile?.length !== 0) {
+    allWidths = allWidths.concat(eligibleWidths?.mobile);
+  }
+  if (userPassedWidths.length !== 0) {
+    allWidths = allWidths.concat(userPassedWidths);
+  } else {
+    allWidths = allWidths.concat(eligibleWidths.config);
+  }
+
+  return [...new Set(allWidths)].filter(e => e); // Removing duplicates
+};
+
+async function changeWindowDimensionAndWait(driver, width, height, resizeCount) {
+  try {
+    if (CDP_SUPPORT_SELENIUM && driver?.capabilities?.browserName === 'chrome') {
```

Comment:
instead we can check in driver responds to executecdpCommand

Reply:
done.